.gitignore

```diff
@@ -1,4 +1,5 @@
 valcord/
+valenv/
```

Comment:
no, dont put your env in the repo in the first place.  if you really feel you want to do that, put this in your personal global exclude file